### PR TITLE
Update init genesis to add default data source and oracle script

### LINKED ISSUE #667: Allow specifying genesis data sources and genesis oracle scripts
Basically implement `InitGenesis` for zoracle.

### LINKED ISSUE #669: Allow bandd init to generate genesis with builtin data sources / oracle scripts
Add a flag to enable that. The list of genesis data sources / oracle scripts can be hardcoded in the source code itself. We can make them configurable later. The flag should also take the address that will be the owner of all these things.

### PR DESCRIPTION
**Pull request by taobun**
fixed #667 
fixed #669

### CODE REVIEW

File: chain/app/genesis.go [hunk 1]
```diff
@@ -119,3 +120,51 @@ func NewDefaultGenesisState() GenesisState {
...
+	dataSources := []struct {
+		name string
+		path string
+	}{
+		{"Coingecko script", "./datasources/coingecko_price.sh"},
+		{"Crypto compare script", "./datasources/crypto_compare_price.sh"},
+		{"Binance price", "./datasources/binance_price.sh"},
+	}
+
+	// TODO: Find a better way to specific path to data sources
```
Comment (sorawit) [inline chain/app/genesis.go]: specify

File: chain/app/genesis.go [hunk 2]
```diff
@@ -119,3 +120,51 @@ func NewDefaultGenesisState() GenesisState {
...
+		}
+		state.DataSources[i] = zoracle.NewDataSource(
+			owner,
+			dataSource.name,
+			sdk.Coins{},
+			script,
+		)
+	}
+
+	// TODO: Find a better way to specific path to oracle scripts
```
Comment (sorawit) [inline chain/app/genesis.go]: ditto

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by sorawit — DISMISSED** [review]
Looks great

**Review by evilpeach — DISMISSED** [review]

**Review by sorawit — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Pull request merged**